samples/transient-clients/Core_6/ClientHub/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Owin.Cors;
+using Microsoft.Owin.Hosting;
+using NServiceBus;
+using Owin;
+
+static class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.TransientClients.ClientHub";
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.TransientClients.ClientHub");
+        endpointConfiguration.UseSerialization<JsonSerializer>();
+        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+
+        var transport = endpointConfiguration.UseTransport<MsmqTransport>();
+        var routing = transport.Routing();
+
+        routing.RegisterPublisher(
+            eventType: typeof(StockTick),
+            publisherEndpoint: "Samples.TransientClients.Publisher");
+
+        endpointConfiguration.SendFailedMessagesTo("error");
+
+        endpointConfiguration.Conventions().DefiningEventsAs(t => t == typeof(StockTick));
```

Comment:
This allows the Client (non-NSB) to share the message type with the NServiceBus endpoints.